### PR TITLE
opsui/notifications: add ability to export `Notifications` into CSV

### PR DESCRIPTION
**Pull request by richard-oven-vanti**
As per a client request, now we have the ability to export notifications in CSV format.
The functionality is available for `all` _(under `Notifications` nav option)_ or `per overview page` _(when selecting an available sub-overview page within `Building Overview`)_.

Exporting notifications

Jira: PRJ-115

### CODE REVIEW

**Review by mattnathan — APPROVED** [review]

**Review by mattnathan — CHANGES_REQUESTED** [review]
The CSV that gets downloaded ends up with a header line that looks like this:
` Create  Time , Source , Floor , Zone , Severity , Description , Resolve  Time , Acknowledged , Acknowledged  Time `

Notice the extra spaces around each word and between `Create` and `Time` etc.

File: ui/conductor/src/routes/ops/notifications/useAlertsApi.js [hunk 1]
```diff
@@ -204,6 +206,87 @@ export default function(name, query) {
...
+        return {
+          'Create Time': createTimeString,
+          'Source': item.source,
+          'Floor': item.floor,
+          'Zone': item.zone,
+          'Severity': notifications.severityData(item.severity).text,
+          'Description': item.description,
+          'Resolve Time': resolveTimeString,
+          'Acknowledged': item.acknowledgement ? 'Yes' : 'No',
+          'Acknowledged Time': ackTimeString
```
Comment (mattnathan) [inline ui/conductor/src/routes/ops/notifications/useAlertsApi.js:272]: Is there a reason you aren't including information about who acknowledged the alert?

**Review by mattnathan — CHANGES_REQUESTED** [review]

File: ui/conductor/src/routes/ops/notifications/useAlertsApi.js (hunk 1 above)
Comment (richard-oven-vanti) [reply ui/conductor/src/routes/ops/notifications/useAlertsApi.js:272]: At the moment, I don't think we export acknowledged alerts, but I'll have a look!

**Review by deanredfern26 — APPROVED** [review]
Tested on site, working

**Review by mattnathan — APPROVED** [review]

**Pull request merged**